openedx/core/djangoapps/user_api/management/tests/test_email_opt_in_list.py

```diff
@@ -411,6 +415,8 @@ def _assert_output(self, output, *args, **kwargs):
         # Check data rows
         for user, course_id, opt_in_pref in args:
             self.assertIn({
```

Comment:
Why not `assertEqual`? There is no expectation that there should be other data in the row.